node/chainSimulator/process/processor.go

```diff
@@ -403,3 +445,12 @@ func GeneratePubKeyBitmap(numOfOnes int) []byte {
 
 	return result
 }
+
+func UnsetBitInBitmap(index int, bitmap []byte) error {
+	if len(bitmap) < index/8 {
+		return errors.New("bitmap too short")
```

Comment:
we already have `ErrWrongSizeBitmap` is errors/common

Reply:
fixed